### PR TITLE
Add betterC support - fixes #1636

### LINKED ISSUE #1636: dub test with betterC
### Bug Description
`dub test` doesn't work when `betterC` is in `buildOptions`

### PR DESCRIPTION
**Pull request by etcimon**
This commit results in the `dub test` command being executed correctly when using the `-betterC` dflag.

### CODE REVIEW

File: source/dub/project.d [hunk 1]
```diff
@@ -1916,22 +1916,33 @@ alias allModules = TypeTuple!(
...
+				import core.runtime;
+				import std.exception;
+				Runtime.moduleUnitTester = () => true;
+				enforce(runUnitTests!allModules(new ConsoleTestResultWriter), "Unit tests failed.");
+			}
+		}
 	}
-}
-};
+};
```
Comment (thewilsonator) [inline source/dub/project.d:1948]: please add EOL before EOF

**Comment by WebFreak001** [comment]
in the future please don't work on master, it makes it harder to collaborate on the PRs

**Review by WebFreak001 — APPROVED** [review]
good solution, I also added a test

**Comment by WebFreak001** [comment]
test failure is on master on windows because of long path names, so I'll ignore it

**Pull request merged**